Client/Frontend/Home/Wallpapers/v2/Utilities/WallpaperURLProvider.swift

```diff
@@ -0,0 +1,71 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+
+enum WallpaperURLType {
+    case metadata
+    case imageURL
+}
+
+struct WallpaperURLProvider {
+    // MARK: - Properties
+    enum URLProviderError: Error {
+        case noBundledURL
+        case problemBuildingURL
```

Comment:
Houston we have a problem? What do you think about renaming it to something like `invalidURL` or  just `urlBuilding`?

Reply:
sure.